traffic_control/clients/python/trafficops/trafficops/tosession.py

```diff
@@ -146,10 +146,10 @@ def update_cdn_by_id(self, cdn_id=None, data=None):
 
     NOTE: Only a small subset of the API endpoints are implemented.  More can be implemented as needed.
           See the Traffic Ops API documentation for more detail:
-              https://trafficcontrol.apache.org/docs/latest/development/traffic_ops.html#api
+              https://trafficcontrol.incubator.apache.org/docs/latest/development/traffic_ops.html#api
```

Comment:
I think the correct URL is: https://traffic-control-cdn.readthedocs.io/en/latest/api/index.html